taskfile.yaml

```
@@ -11,8 +11,9 @@ vars:
   G_CPP_SRC_DIR: "{{.ROOT_DIR}}/src"
   G_DEPS_DIR: "{{.G_BUILD_DIR}}/deps"
 
-  # This should be kept in-sync with its usage in CMakeLists.txt
-  G_DEPS_CMAKE_SETTINGS_FILE: "{{.G_DEPS_DIR}}/settings.cmake"
+  # These should be kept in-sync with its usage in CMakeLists.txt
+  G_DEPS_CMAKE_SETTINGS_DIR: "{{.G_DEPS_DIR}}/CMake"
```

Match the CMakeLists.txt change.
```suggestion
  G_DEPS_CMAKE_SETTINGS_DIR: "{{.G_DEPS_DIR}}/cmake-settings"
```